src/components/Context.ts

```diff
@@ -15,34 +15,30 @@ export interface ReactReduxContextValue<
   noopCheck: CheckFrequency
 }
 
-const ContextKey = Symbol.for(`react-redux-context-${ReactVersion}`)
-const gT = globalThis as { [ContextKey]?: Context<ReactReduxContextValue> }
+const ContextKey = Symbol.for(`react-redux-context`)
+const gT = globalThis as {
+  [ContextKey]?: Map<
+    typeof React['createContext'],
+    Context<ReactReduxContextValue>
+  >
+}
+
+function getContext(): Context<ReactReduxContextValue> {
+  if (!React.createContext) return {} as any
```

Comment:
In an environment where `createContext` does not exist we can pretty much just return anything - there is no `useContext` that could consume it anyways.

Reply:
and the provider needs to be in a client component, so presumably that won't be an issue either